<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1737</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting Algorithm Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>1561</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowminimumheight="0" columnminimumwidth="0"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="sortButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Merge Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="numBarsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Number of Elements:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNum">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>40</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1737</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSorting_Algorithms">
    <property name="title">
     <string>Algorithms</string>
    </property>
    <addaction name="actionMerge_Sort"/>
    <addaction name="actionBubble_Sort"/>
    <addaction name="actionRadix_Sort"/>
    <addaction name="actionQuick_Sort"/>
    <addaction name="actionSelection_Sort"/>
   </widget>
   <addaction name="menuSorting_Algorithms"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMerge_Sort">
   <property name="text">
    <string>Merge Sort</string>
   </property>
  </action>
  <action name="actionBubble_Sort">
   <property name="text">
    <string>Bubble Sort</string>
   </property>
  </action>
  <action name="actionRadix_Sort">
   <property name="text">
    <string>Radix Sort</string>
   </property>
  </action>
  <action name="actionQuick_Sort">
   <property name="text">
    <string>Quick Sort</string>
   </property>
  </action>
  <action name="actionSelection_Sort">
   <property name="text">
    <string>Selection Sort</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
